frontend/src/features/tasks/components/interactive/InteractiveFormField.tsx

```
@@ -0,0 +1,216 @@
+                  <Badge variant="secondary" className="text-xs">
+                    Recommended
+                  </Badge>
+                )}
+              </Label>
+            </div>
+          ))}
+        </RadioGroup>
+      )
+
+    case 'multiple_choice':
+      const selectedValues = (value as string[]) ?? []
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)